src/calculateRentalCost.js

```diff
@@ -27,8 +27,27 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
+ function calculateRentalCost(days) {
+  let cost = 0;
+  const price = 40;
+  const discountMore7 = 50;
```

Comment:
```suggestion
  const bigDiscount = 50;
```
Плохая практика привязывать названия переменных к чему-то. Если у тебя скидка будет не через 7 дней, а через 8, то нужно будет переименовывать переменную

Reply:
Понял, спасибо